### DISCUSSION TITLE
关于中国中产的定义

### DISCUSSION THREAD

**Discussion by Lewisrxliu** [RoadSideUniversity/ClubHouse2024#10] [Q&A]
https://github.com/RoadSideUniversity/-ClubHouse2023Nov/assets/128250203/804b0608-a5b3-4fd0-ab10-05f81baca3f8

**Comment by guttasingapore** [comment]
新(伪)中产的阶层困境 https://zhuanlan.zhihu.com/p/83173302
中产要么向上突破，要么向下自由落体… https://xueqiu.com/5218225681/251277694
国歌宣泄不满 官方径直封禁 https://www.rfi.fr/cn/中国/20220420-上海有人唱国歌宣泄对封城不满-官方径直封禁-起来-词
国际歌为什么现在是禁歌 https://www.douyin.com/shipin/7261632387174467642

**Reply by Lewisrxliu** [reply]
中国没有老龄化危机
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/82bda976-25af-4e27-acb0-5737039e6aec)

**Reply by guttasingapore** [reply]
网易《如此打工30年》- 中国底层 2024.1.8
https://x.com/whyyoutouzhele/status/1744620724592037956?s=20

**Reply by Lewisrxliu** [reply]
看中国人口问题的不同角度
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/32541718-b91f-4aa4-b57e-3b0899c03aef)

**Comment by Lewisrxliu** [comment]
> https://github.com/RoadSideUniversity/-ClubHouse2023Nov/assets/128250203/804b0608-a5b3-4fd0-ab10-05f81baca3f8

详解如下
https://www.163.com/dy/article/IN6MM7PO0553XH85.html

**Reply by Lewisrxliu** [reply]
茅于轼
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/6140076a-e786-443e-9617-e9380f00eee8)

**Comment by Lewisrxliu** [comment]
大学沙龙（原哈佛沙龙）第198期
 罗斯高（Scott Rozelle）—— 中国农村的共同富裕：城乡差距如何威胁中国的崛起

■ 时间与报名信息
 
美东时间：2024年01月20日 周六晚上8点
北京时间：2024年01月21日 周日早上9点
 
zoom 报名链接

https://clt1245285.benchurl.com/c/l?u=10A17FFC&e=177E3F1&c=130065&t=0&l=7629A759&email=wEudc9%2BnBW78ss29WFaww5q4wOJ2jUnu&seq=1

**Comment by Lewisrxliu** [comment]
大头针社会
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/632046d5-c35c-45b7-89e2-f9d200fd2917)

**Reply by Lewisrxliu** [reply]
许成钢的有救吗？
https://www.bumingbai.net/2024/02/ep-084-xu-chenggang-chinese-economy-2024-text/
https://www.rfi.fr/cn/中国/20240211-学者称中国经济成长5-2-不可靠-或仅有0和1-之间
https://twitter.com/BingLiu34173809/status/1760123400914411679

**Reply by Lewisrxliu** [reply]
许成钢推荐了一本书：The Code of Capital: How the Law Creates Wealth and Inequality 《财富背后的法律密码-法律如何创造财富与不平等》 卡塔琳娜·皮斯托(Katharina Pistor), 2019 
https://youtu.be/E7vuceViUCA?si=rQ5m2PxUKtdGJ0N3
简介
https://m.thepaper.cn/newsDetail_forward_10044043?from=
https://cifu.fudan.edu.cn/_upload/article/files/dc/6a/cdc599ac4db38b9dc63c87b9927a/20f75f05-42c6-470f-bed1-2af1b3aac3cf.pdf

**Reply by Lewisrxliu** [reply]
关于许成钢
Xiaoping曾经问过我一个问题：中国到底有没有知识分子。强调一下时间地点 - 目前，在国内，之前我的回答不变。
许成钢的父亲许良英是束星北和王淦昌的学生，这么理解就比较容易了。
https://zh.m.wikipedia.org/zh-hans/许成钢
https://zh.m.wikipedia.org/wiki/许良英
许成钢的个人主页
https://www.cgxuwebsite.com
